app/Models/User.php

```diff
@@ -45,4 +45,8 @@ public function bots()
     {
         return $this->hasMany(Bot::class);
     }
+    
+    public function webhooks() {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/User.php</br>**Line**: 49</br>Opening brace should be on a new line